grader_labextension/handlers/config.py

```diff
@@ -0,0 +1,24 @@
+from tornado.web import HTTPError, authenticated
+
+from grader_labextension.handlers.base_handler import ExtensionBaseHandler
+from grader_labextension.registry import register_handler
+from grader_labextension.services.request import RequestServiceError
+
+
+@register_handler(path=r"api\/config\/?")
+class ConfigHandler(ExtensionBaseHandler):
+    @authenticated
+    async def get(self):
+        try:
+            # see setup_handlers() and get_grader_config() in __init__.py of grader_labextension
+            data = self.settings["page_config_data"]
+            cell_timeout_keys = ["default_cell_timeout", "min_cell_timeout", "max_cell_timeout"]
+            cell_timeout_data = {k: data[k] for k in cell_timeout_keys}
+            self.log.info("Retrived config values for cell timeout " + str(cell_timeout_data))
```

Comment:
Just a nit-pick, but a better way to format a string is to use an [`f-string`](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings):

```suggestion
            self.log.info(f"Retrived config values for cell timeout {cell_timeout_data}")
```
This will automatically convert `cell_timeout_data` into a string that can be inserted into the whole template.

Or, even better - since f-strings are always evaluated, and [log messages are supposedly only evaluated when they are logged](https://realpython.com/python-logging/#displaying-variable-data) (probably saves us a few nanoseconds, so not really important, but just as a fun-fact, with a very specific definition of "fun"):

```suggestion
            self.log.info("Retrived config values for cell timeout %s", cell_timeout_data)
```
In this case, [the first parameter is a message format string](https://docs.python.org/3/library/logging.html#logging.Logger.debug), and whatever additional arguments you provide, will be inserted into the template, replacing `%s`.